server/ai_process.go

```diff
@@ -1520,6 +1521,24 @@ func processAIRequest(ctx context.Context, params aiRequestParams, req interface
 			break
 		}
 
+		// Don't suspend the session if the error is a transient error.
+		if isRetryableError(err) {
+			if cap == core.Capability_LiveVideoToVideo {
+				params.sessManager.Complete(ctx, sess)
+				continue
+			} else {
+				//keep session out of selection on this request for batch AI jobs
```

Comment:
Could you explain (in the code comment) the reasoning why you want to keep it out for this request in case of batch AI Jobs?

Reply:
I thought about this some more and removed this if stmt to go back to calling `Complete` on the session for batch AI as well.  The retryable error could be request specific and the Orchestrator session may work on another concurrent request.  We achieve the main main goal to not suspend an Orchestrator for these errors and if find it to be spamming the Orchestrators for batch jobs will re-add in separate PR.